### PR TITLE
Location inputs

### PR DESCRIPTION
**Pull request by Andrew-Nes**
# What type of PR is this? (check all applicable)
 🍕 Feature

Add location input, to be integrated with user data next PR

### CODE REVIEW

File: pet-finder/src/Components/LocationInputs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { FC, useState } from "react";
+
+import GooglePlacesAutocomplete from 'react-google-places-autocomplete'; 
+
+import * as LocationInterfacres from '../../types/location.type';
```
Comment (RazumauMikita) [inline pet-finder/src/Components/LocationInputs/index.tsx]: Mistyping in import name:
```
 LocationInterfacres
```

File: pet-finder/src/Components/LocationInputs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const lang = 'en';
+
+
+export const LocationInput: FC = () => {
+const [location, setLocation] = useState<LocationInterfacres.Country | null>(null);
+console.log(location);
+  return(
+  <div>
+    <GooglePlacesAutocomplete
+      // apiKey={} TODO add API key fron env
```
Comment (RazumauMikita) [inline pet-finder/src/Components/LocationInputs/index.tsx]: There will be import from env: 
```
apiKey={import.meta.env.VITE_GOOGLE_MAP_API_KEY}
```

File: pet-finder/src/Components/LocationInputs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { FC, useState } from "react";
+
```
Comment (RazumauMikita) [inline pet-finder/src/Components/LocationInputs/index.tsx]: These imports might be within one group: 
```
import { FC, useState } from "react";
import GooglePlacesAutocomplete from 'react-google-places-autocomplete';
```

Comment (Andrew-Nes) [reply pet-finder/src/Components/LocationInputs/index.tsx]: fixed

File: pet-finder/src/Components/LocationInputs/index.tsx (hunk 2 above)
Comment (Andrew-Nes) [reply pet-finder/src/Components/LocationInputs/index.tsx]: fixed

File: pet-finder/src/Components/LocationInputs/index.tsx (hunk 1 above)
Comment (Andrew-Nes) [reply pet-finder/src/Components/LocationInputs/index.tsx]: fixed

**Pull request merged**